core/src/test/scala/unit/kafka/server/FetchRequestDownConversionConfigTest.scala

```diff
@@ -221,6 +222,8 @@ class FetchRequestDownConversionConfigTest extends BaseRequestTest {
     } else {
       assertEquals(Errors.UNSUPPORTED_VERSION, error(partitionWithDownConversionDisabled))
     }
+    TestUtils.waitUntilTrue(() => TestUtils.metersCount(BrokerTopicStats.FetchMessageConversionsPerSec) > initialFetchMessageConversionsPerSec,
+    s"init: $initialFetchMessageConversionsPerSec final: ${TestUtils.metersCount(BrokerTopicStats.FetchMessageConversionsPerSec)}", 3000)
```

Comment:
nit: The error message could be more clear, ex:
```
The `FetchMessageConversionsPerSec` metric count is not incremented after 3 seconds. init: $initialFetchMessageConversionsPerSec final: ${TestUtils.metersCount(BrokerTopicStats.FetchMessageConversionsPerSec)}
```

Also, the 3 seconds might be able to increase to 5 sec to avoid flaky failure.

Reply:
thanks for review. will address your comment